docs/source/release_notes.rst

```diff
@@ -10,12 +10,13 @@ Future Release
     * Fixes
     * Changes
         * Entirely null columns are now inferred as the Unknown logical type (:pr:`1043`)
+        * ``TableAccessor.select`` method will now maintain dataframe column ordering (:pr:`1052`)
```

Comment:
should this specify "maintain dataframe column ordering when ``return_schema` is True" or something?

Reply:
How about: `...maintain dataframe column ordering in TableSchema columns?`

Reply:
that sounds good to me!